# 3.2. Python exceptions

